src/PageDocumentation/index.tsx

```diff
@@ -203,7 +203,7 @@ function ArtifactSetKeyPane() {
 }
 function CharacterKeyPane() {
   const { t } = useTranslation([...new Set(allCharacterKeys)].map(k => `char_${k}_gen`))
-  const charKeysCode = `type CharacterKey\n  = ${[...new Set(allCharacterKeys)].sort().map(k => `"${k}" //${t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
+  const charKeysCode = `type CharacterKey\n  = ${[...new Set(locationCharacterKeys)].sort().map(k => `"${k}" //${k === "Traveler" ? "Traveler" : t(`char_${k}_gen:name`)}`).join(`\n  | `)}`
```

Comment:
Use `charNames_gen.json` so we don't have to load every character's translation json

Reply:
While we are at it, we could also update the artifact and weapons to use `artifactNames_gen.json` and `weaponNames_gen.json`